frontend/src/pages/projects/screens/detail/storage/BaseStorageModal.tsx

```diff
@@ -119,7 +119,7 @@ const BaseStorageModal: React.FC<BaseStorageModalProps> = ({
               editableK8sName={!existingPvc}
             />
           </StackItem>
-          {children}
+          {typeof children === 'function' ? children(createData.name) : children}
```

Comment:
@Gkrumbach07 `getDisplayNameFromK8sResource` works perfectly fine when we have `existing cluster storage` but for creating new Cluster storage, this will return `undefined`